packages/contentful--create-contentful-app/lib/index.js

```diff
@@ -60,29 +62,25 @@ function initProject() {
       process.exit(1);
     }
 
-    const initCommand = 'node';
-    const createReactApp = require.resolve('create-react-app');
-    const template = getTemplate();
-
-    const args = [createReactApp, appFolder, '--template', template, '--use-npm'];
+    console.log(`Creating a Contentful app in ${chalk.bold(tildify(fullAppFolder))}.`);
 
-    console.log(
-      `Creating a Contentful app in ${chalk.bold(tildify(path.resolve(process.cwd(), appFolder)))}.`
+    await cloneTemplate(
+      'typescript#ext-3435', // TODO: needs to be adjusted once PR is merged
+      fullAppFolder
     );
 
-    const appCreateProcess = spawn(initCommand, args, { stdio: 'inherit' });
-    appCreateProcess.on('exit', (exitCode) => {
-      if (exitCode === 0) {
-        onSuccess(appFolder);
-      } else {
-        process.exit(exitCode);
-      }
-    });
+    rmIfExists(resolve(fullAppFolder, 'package-lock.json'));
+    rmIfExists(resolve(fullAppFolder, 'yarn.lock'));
+    updatePackageName(fullAppFolder);
+
+    await exec('npm', ['install'], { cwd: fullAppFolder });
```

Comment:
nit: maybe we can check if npm version is >=6 as we require in the package json? I am not sure if we get any warnings or anything when executed as child process

Reply:
The `engines` property in package.json of `create-contentful-app` should show a warning like this:

```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'create-contentful-app@1.0.0',
npm WARN EBADENGINE   required: { node: '>=16.0.0' },
npm WARN EBADENGINE   current: { node: 'v14.15.0', npm: '7.5.3' }
npm WARN EBADENGINE }
```

Reply:
Yeah, I was more thinking about an explicit check for wen we incorporate yarn, for example. We can handle that then though

Reply:
Yes, we might want to check whether `yarn` is installed. And even when using `yarn`, we probably still have a minimum npm version.
Let's do this in a future PR.